modules/core/src/main/java/org/apache/synapse/aspects/flow/statistics/collectors/CallbackStatisticCollector.java

```diff
@@ -48,6 +49,11 @@ public static void addCallback(MessageContext messageContext, String callbackId)
 
 			CallbackSentEvent callbackSentEvent = new CallbackSentEvent(dataUnit);
             addEventAndIncrementCallbackCount(messageContext, callbackSentEvent);
+
+            if (isOpenTracingEnabled()) {
```

Comment:
Looks formatting is not correct. Go through all and reformat
Use 4 spaces for indentation (it should mat the existing file. Existing code use 4 spaces for indentation)

Reply:
Done in https://github.com/wso2/wso2-synapse/pull/1466/commits/f667809fdde325456df56a477b83e6ef731107ca